cmd/gitserver/server/vcs_syncer_git.go

```diff
@@ -2,13 +2,30 @@ package server
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"os/exec"
 
 	"github.com/sourcegraph/sourcegraph/internal/vcs"
 	"github.com/sourcegraph/sourcegraph/lib/errors"
 )
 
+// GitCommandError is a Git command that encountered an error.
```

Comment:
probably more like this?
```suggestion
// GitCommandError is an error of a failed Git command.
```

I mean at least it sounded like it is the command itself initially :)